falcon/util/uri.py

```diff
@@ -26,10 +26,8 @@
 
 from falcon.constants import PYPY
 try:
-    from falcon.cyutil.uri import (
-        decode as _cy_decode,
-        parse_query_string as _cy_parse_query_string,
-    )
+    from falcon.cyutil.uri import decode as _cy_decode
+    from falcon.cyutil.uri import parse_query_string as _cy_parse_query_string
```

Comment:
You'll need to either revert, or unroll the `except ImportError` clause for both statements, otherwise one is missed in coverage (as we are not aggregating coverage from tests against the cythonized flavour).